tests/test.linter.js

```diff
@@ -114,31 +105,29 @@ describe('Linter', () => {
     expect(output.summary.warnings).toEqual(0);
   });
 
-  it('should collect an error when not an xpi/zip', () => {
+  it('should collect an error when not an xpi/zip', async () => {
     const addonLinter = new Linter({ _: ['tests/fixtures/not-a-zip.zip'] });
     // Stub print to prevent output.
     addonLinter.print = sinon.stub();
     expect(addonLinter.collector.errors.length).toEqual(0);
-    return addonLinter.scan()
-      .catch(() => {
-        expect(addonLinter.collector.errors.length).toEqual(1);
-        expect(addonLinter.collector.errors[0].code).toEqual(
-          messages.BAD_ZIPFILE.code);
-      });
+    await expect(
+      addonLinter.scan()
+    ).rejects.toThrow();
```

Comment:
@hameleonka we should change these empty `await expect(...).rejects.toThrow()` assertions (this one and the other ones in this pull request) into an assertion which includes a proper assertion of the expected error message (otherwise we may miss regressions related to unexpected errors raised instead of the expected one).